firmware/smbios/smbios.ksy

```diff
@@ -0,0 +1,37 @@
+meta:
+  id: smbios
+  endian: le
+  imports:
+    - smbios_bios_info
+    - smbios_system_info
+    - smbios_processor_info
+    - smbios_portable_battery
+    - smbios_skipper
+    - smbios_strings
+seq:
+  - id: tables
+    type: table
+    repeat: eos
+types:
+  table:
+    seq:
+      - id: type
+        type: u1
+        enum: type_enum
```

Comment:
From what I've observed so far, it looks like all SMBIOS tables have common header parts, like length + handle. Why not parse them here, along with `type`?